source/voice_control/about_wake_word.markdown

```diff
@@ -78,9 +78,30 @@ How wake words integrate into Home Assistant
 
 As an example, we’re also making the **Porcupine (v1)** wake word engine available. It supports 29 wake words across English, French, Spanish, and German. The wake words include *Computer*, *Framboise*, *Manzana*, and *Stachelschwein*.
 
+## About on-device wake word processing (microWakeWord)
+
+The [microWakeWord] created by [Kevin Ahrendt] enables ESPHome to detect wake words on devices like the ESP32-S3-BOX-3.
+
+Because openWakeWord is too large to run on low-power devices like the S3-BOX-3, openWakeWord runs wake word detection on the Home Assistant server.
+
+<p><img src='/images/assist/assist_open-vs-micro-wake-word.png' class='no-shadow' /></p>
+
+Doing wake word detection on Home Assistant allows low-power devices like the [M5 ATOM Echo Development Kit][m5-tutorial] to simply stream audio and let all of the processing happen elsewhere.
+The downside is that adding more voice assistants requires more CPU usage in Home Assistant as well as more network traffic.
+
```

Comment:
Not sure how to review that, but all links on this page seem to be failing (See the brackets everywhere)
![CleanShot 2024-02-21 at 15 30 22](https://github.com/home-assistant/home-assistant.io/assets/5878296/c87792f2-7e01-410b-9f3e-21ed697afcd1)